scripts/enforce-call-super-for-overridden-onupdate

```diff
@@ -5,7 +5,9 @@ set -e -o pipefail -u
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
-if git grep --files-with-matches 'void onUpdate' -- '*.java' ':!*/BaseProjectKeyAvailabilityAssuranceAction.java' | xargs git grep --files-without-match 'extends BaseProjectKeyAvailabilityAssuranceAction\|super\.onUpdate'
+if git grep --files-with-matches 'void onUpdate' -- '*.java' ':!*/BaseProjectKeyAvailabilityAssuranceAction.java' |
```

Comment:
Nit: I'd rather move `| ` to the following line (and add `\` here at the EOL) so that it's clear that the next line is a part of the pipeline and not a standalone statement